nuxt.config.js

```diff
@@ -1,84 +1,93 @@
+        { link: [{ rel: 'icon', type: 'image/svg+xml', href: "/icon_share_twin.svg" }] }
+      ]
+    }
   },
 
-  // Global CSS: https://go.nuxtjs.dev/config-css
-  css: [
-  ],
-
-  // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
-  plugins: [
-    '@/plugins/axios'
-  ],
-
-  // Modules for dev and build (recommended): https://go.nuxtjs.dev/config-modules
-  buildModules: [
-    // https://go.nuxtjs.dev/eslint
-    '@nuxtjs/eslint-module',
-    // https://go.nuxtjs.dev/vuetify
-    '@nuxtjs/vuetify'
-  ],
+  ssr: false,
+  target: 'static',
 
-  // Modules: https://go.nuxtjs.dev/config-modules
-  modules: [
-    // https://go.nuxtjs.dev/axios
-    '@nuxtjs/axios',
-    '@nuxtjs/recaptcha'
-  ],
+  // ** Customize the progress-bar color
+  loading: { color: '#fff' },
 
-  recaptcha: {
-    version: 2,
-  },
+  // ** Global CSS
+  css: ['vuetify/lib/styles/main.sass'],
 
-  // Axios module configuration: https://go.nuxtjs.dev/config-axios
-  axios: {},
+  // ** Nuxt.js modules
+  modules:
+    [
+      // ['@dargmuesli/nuxt-cookie-control',
+      //   {
+      //     controlButton: true,
+      //     domain: 'geode-solutions.com',
+      //     colors: {
+      //       modalOverlay: '#000',
+      //       barBackground: colors.teal.darken1,
+      //       barButtonColor: '#000',
+      //       modalTextColor: '#000',
+      //       modalOverlayOpacity: 0,
+      //       modalButtonColor: '#fff',
+      //       modalUnsavedColor: '#000',
+      //       modalButtonBackground: colors.teal.darken1,
+      //       controlButtonIconColor: colors.teal.darken1,
+      //       checkboxActiveBackground: colors.teal.darken1,
+      //       checkboxInactiveBackground: '#000',
+      //       modalButtonHoverBackground: '#333',
+      //       checkboxDisabledBackground: '#ddd',
+      //       controlButtonIconHoverColor: colors.amber.accent4,
+      //       controlButtonHoverBackground: colors.brown.darken4
+      //     }
+      //   }
+      // ],
+      ['@pinia/nuxt',
+        {
+          autoImports: [
+            'storeToRefs',
+          ],
+        }],
+    ],
 
-  // Vuetify module configuration: https://go.nuxtjs.dev/config-vuetify
-  vuetify: {
-    theme: {
-      themes: {
-        light: {
-          primary: colors.teal.darken1,
-          secondary: colors.teal.lighten4,
-          accent: colors.red.darken4
-        }
+  cookies: {
+    necessary: [
+      {
+        name: 'Default Cookies',
+        description: 'Used for cookie control.',
+        cookies: ['cookie_control_consent', 'cookie_control_enabled_cookies'],
+        isSecureContext: true
       }
-    },
-    icons: { iconfont: 'mdi', values: { logo: { component: 'GeodeLogo' } } }
+    ]
   },
 
-  // Build Configuration: https://go.nuxtjs.dev/config-build
+  // ** Build configuration
   build: {
+    transpile: ['vuetify'],
+  },
+  vue: {
+    compilerOptions: {
+      isCustomElement: (tag) => ['md-linedivider'].includes(tag)
```

Comment:
Where is this used?

Reply:
Nuxt is having trouble identifying which component is local and which is from vuetify, so this is used for every vuetify component